sos/report/plugins/infinidat.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2024 Alejandro Santoyo <alejandro.santoyo@canonical.com>
+#
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+
+import platform
+from sos.report.plugins import Plugin, IndependentPlugin
+    
+    
+class InfinidatStoragePlugin(Plugin, IndependentPlugin):
+    
+    short_desc = 'Infinidat Storage plugin'
+    plugin_name = 'infinidat'
+    profiles = ('storage')
```

Comment:
needs t o be a tuple, so add a comma after `'storage'`

Reply:
Fixed, thanks